src/main/java/Duke.java

```diff
@@ -1,10 +1,331 @@
+                        }
+                    }
+                    break;
+                case "deadline":
+                    try {
+                        if (actionArr.length == 1) {
+                            throw new DukeException("OOPS!!! Deadline description cannot be empty.");
+                        }
+                    } catch (DukeException c){
+                        System.out.println(c);
+                        break;
+                    }
+                    String[] deadlineParts = actionArr[1].split("/by");
+                    try {
+                        if (deadlineParts.length == 1) {
+                            throw new DukeException("OOPS!! You did not include a deadline using /by. :(");
+                        }
+                    } catch (DukeException d){
+                        System.out.println(d);
+                        break;
+                    }
+                    Deadline d = new Deadline(deadlineParts[0], deadlineParts[1]);
+                    allTasks.add(d);
+                    System.out.println("----------------------------------------------------");
+                    System.out.println("Got it. I've added this task:");
+                    d.toString();
+                    System.out.println("Now you have "+ allTasks.size() +" tasks in the list.");
+                    System.out.println("----------------------------------------------------");
+                    taskNum = allTasks.size();
+                    if (taskNum==1) {
+                        try {
+                            writeToFile(filePath, taskNum + ". " + allTasks.get(taskNum-1).toString() + "\n");
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+                    } else {
+                        try {
+                            appendToFile(filePath, taskNum + ". " + allTasks.get(taskNum-1).toString() + "\n");
+                        } catch (IOException e) {
+                            System.out.println("Something went wrong: " + e.getMessage());
+                        }
+                    }
+                    break;
+                case "event":
+                    try {
+                        if (actionArr.length == 1) {
+                            throw new DukeException("OOPS!!! Event description cannot be empty.");
+                        }
+                    } catch (DukeException e){
+                        System.out.println(e);
+                        break;
+                    }
+                    String[] eventParts = actionArr[1].split("/at");
+                    try {
+                        if (eventParts.length == 1) {
+                            throw new DukeException("OOPS!! You did not include an event date using /at. :(");
+                        }
+                    } catch (DukeException z){
```

Comment:
A small indentation error here. According to our coding standards, there is no indentation for case clauses. See [Coding Standards](https://se-education.org/guides/conventions/java/basic.html#conditionals)